libcxx/include/__functional/reference_wrapper.h

```diff
@@ -64,6 +67,62 @@ class _LIBCPP_TEMPLATE_VIS reference_wrapper : public __weak_result_type<_Tp> {
   {
     return std::__invoke(get(), std::forward<_ArgTypes>(__args)...);
   }
+
+#if _LIBCPP_STD_VER >= 26
+
+  // [refwrap.comparisons], comparisons
+
+  friend constexpr bool operator==(reference_wrapper __x, reference_wrapper __y)
+    requires requires {
+      { __x.get() == __y.get() } -> __boolean_testable;
+    }
+  {
+    return __x.get() == __y.get();
+  }
+
+  friend constexpr bool operator==(reference_wrapper __x, const _Tp& __y)
+    requires requires {
+      { __x.get() == __y } -> __boolean_testable;
+    }
+  {
+    return __x.get() == __y;
+  }
+
+  friend constexpr bool operator==(reference_wrapper __x, reference_wrapper<const _Tp> __y)
+    requires(!is_const_v<_Tp>) && requires {
+      { __x.get() == __y.get() } -> __boolean_testable;
+    }
+  {
+    return __x.get() == __y.get();
+  }
+
+  // `operator<=>`: Checks the constraints of `synth-three-way` as per https://wg21.link/LWG4071 directly
+
+  friend constexpr auto operator<=>(reference_wrapper __x, reference_wrapper __y)
+    requires requires(const _Tp t) {
+      { t < t } -> __boolean_testable;
```

Comment:
```suggestion
    requires requires(const _Tp __t) {
      { __t < __t } -> __boolean_testable;
```
However I wonder why this constraint is used. It does not match LWG 4071.

Reply:
See https://github.com/gcc-mirror/gcc/commit/d86472a6f041ccf3d1be0cf6bb15d1e0ad8f6dbe. This approach is also suggested [LWG4071](https://cplusplus.github.io/LWG/issue4071) as an equivalent but simpler implementation stategy.

Reply:
```suggestion
  // `operator<=>`: Checks the constraints of `synth-three-way` as per https://wg21.link/LWG4071 directly
```
The comment on line 99 is  supposed to explain just that the contraint is implemented according to the recommendation in [LWG4071](https://cplusplus.github.io/LWG/issue4071), obviously I failed to do that.

Reply:
>  In practice the requires-clause can be implemented more simply (and efficiently) by checking the constraints of synth-three-way directly:
>
>    `requires (const T t) { { t < t } -> boolean-testable; }`
>
>but when specified in prose in a Constraints: element it seems clearer to just use synth-three-way(x.get(), y.get()).